src/gamma.rs

```diff
@@ -59,6 +59,9 @@ where
 
     /// Compute the natural logarithm of the gamma function.
     fn ln_gamma(self) -> (Self, i32);
+
+    /// Compute the trigamma function of `x`.
```

Comment:
For consistency, please remove “of ‘x’”.

Reply:
Do you have a reference for this implementation? Look at the descriptions above. It would be nice to indicate what the implementation is based on.

Reply:
I have copy-pasted the implementation from Julia's SpecialFunction module. They cite some kind of textbook over there, but I have not read it, just verified the output of their implementation with a set of scipy.special.polygamma(1,...) calls and a few special cases from wikipedia.

Reply:
I think a simple citation to this module would be enough. Please take a look at how it’s written for other functions. Thank you!

Reply:
Added the link